features/__snapshots__/validate_image.snap

```diff
@@ -416,3 +416,11 @@ Error: success criteria not met
       other: data2
 
 ---
+
+[image config:stdout - 1]
+{"success":true,"components":[{"name":"Unnamed","containerImage":"${REGISTRY}/acceptance/image-config@${REGISTRY_acceptance/image-config:latest_HASH}","successes":[{"msg":"Pass","metadata":{"code":"builtin.attestation.signature_check","title":"Attestation signature check passed"}},{"msg":"Pass","metadata":{"code":"builtin.attestation.syntax_check","title":"Attestation syntax check passed"}},{"msg":"Pass","metadata":{"code":"builtin.image.signature_check","title":"Image signature check passed"}},{"msg":"Pass","metadata":{"code":"image_config.image_distinct_title_set","description":"Check if the image has a different value than its parent image for the org.opencontainers.image.title label.","title":"Image Distinct Title Label"}},{"msg":"Pass","metadata":{"code":"image_config.image_title_set","description":"Check if the image has the org.opencontainers.image.title label set.","title":"Image Title Label"}},{"msg":"Pass","metadata":{"code":"image_config.parent_image_title_set","description":"Check if the parent image has the org.opencontainers.image.title label set.","title":"Parent Image Title Label"}}],"success":true,"signatures":[${IMAGE_SIGNATURES_JSON_acceptance/image-config}],"attestations":[{"type":"https://in-toto.io/Statement/v0.1","predicateType":"https://slsa.dev/provenance/v0.2","predicateBuildType":"https://tekton.dev/attestations/chains/pipelinerun@v2","signatures":[${ATTESTATION_SIGNATURES_JSON_acceptance/image-config}]}]}],"key":${known_PUBLIC_KEY_JSON},"policy":{"sources":[{"policy":["git::https://${GITHOST}/git/image-config-policy.git"]}],"rekorUrl":"${REKOR}","publicKey":"${known_PUBLIC_KEY}"},"ec-version":"${EC_VERSION}","effective-time":"${TIMESTAMP}"}
```

Comment:
Can we use that new trick to make this indented and formatted? Not sure why it's not doing that already, the code seems to be towards the end of MatchSnapshot in acceptance/snaps/snaps.go .

Reply:
The JSON format is broken with the variable substitutions; one example search for `${ATTESTATION_SIGNATURES_JSON`. See [HACBS-2337](https://issues.redhat.com/browse/HACBS-2337)

Reply:
Oooh, thanks. Now I understand what that [HACBS-2337](https://issues.redhat.com/browse/HACBS-2337) is about..